launch/tier4_vehicle_launch/urdf/model.xacro

```diff
@@ -2,12 +2,14 @@
 <robot name="vehicle" xmlns:xacro="http://ros.org/wiki/xacro">
   <xacro:arg name="vehicle_model" default=""/>
   <xacro:arg name="sensor_model" default=""/>
+  <xacro:arg name="config_dir" default="$(find $(arg sensor_model)_description)/config"/>
```

Comment:
Do we have to add default argument here?
The reason I feel it better to remove the default argument is here
https://github.com/autowarefoundation/autoware.universe/pull/1337#discussion_r928199947

Reply:
That makes sense. The default value was already used in sensor.xacro :)